src/realm/table_view.cpp

```diff
@@ -185,84 +191,96 @@ R TableViewBase::aggregate(R (ColType::*aggregateMethod)(size_t, size_t, size_t,
     typedef typename ColTypeTraits::leaf_type ArrType;
     const ColType* column = static_cast<ColType*>(&m_table->get_column_base(column_ndx));
 
+    // FIXME: Optimization temporarely removed for stability
+/*
```

Comment:
I would prefer that you remove all this optimisation code instead of commenting it out, because it could take some larger redesign to optimise again and it will most likely be in a different way. But it's not blocking for me.